docs/UserGuide.md

```diff
@@ -41,153 +50,116 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
+Creates a new event  
+Format: `newevent`  
+Creates a new event with the specified event **NAME**  
+Example: 
+- `newevent Singing` creates an event that has the name “Singing”.
+
+Listing all events : listevent
+Shows a list of all events in the address book
+Format: listevent
+
+
+Deleting an event : deleteevent
+Deletes the specified event from the event book.
+Format: deleteevent INDEX
+Deletes the event at the specified INDEX
+The index refers to the index number shown in the displayed event list.
+The index must be a positive integer 1, 2, 3, …​
 Examples:
-* `list` followed by `delete 2` deletes the 2nd person in the address book.
-* `find Betsy` followed by `delete 1` deletes the 1st person in the results of the `find` command.
+list followed by delete 2 deletes the 2nd event in the event book.
 
-### Clearing all entries : `clear`
 
-Clears all entries from the address book.
+Adds dates to an event : adddate
+Adds a starting time and ending time to the specified event in the event book.
+Format: adddate INDEX /from dd/MM/yy /to dd/MM/yy
+Adds starting and ending time to the event at the specified INDEX
+The index refers to the index number shown in the displayed event list.
+The index must be a positive integer 1, 2, 3, …​
+The event must not have time associated with it beforehand
 
-Format: `clear`
 
-### Exiting the program : `exit`
+Saving all events and contacts: save
+Saves all entries in Paidlancer.
+Format: save
 
-Exits the program.
 
-Format: `exit`
+Loading all events and contacts : load
+Loads all entries in Paidlancer from local storage if it exist
+Format: load
 
 ### Saving the data
 
-AddressBook data are saved in the hard disk automatically after any command that changes the data. There is no need to save manually.
-
-### Editing the data file
-
-AddressBook data are saved as a JSON file `[JAR file location]/data/addressbook.json`. Advanced users are welcome to update data directly by editing that data file.
-
-<div markdown="span" class="alert alert-warning">:exclamation: **Caution:**
-If your changes to the data file makes its format invalid, AddressBook will discard all data and start with an empty data file at the next run.
-</div>
-
-### Archiving data files `[coming in v2.0]`
-
-_Details coming soon ..._
-
---------------------------------------------------------------------------------------------------------------------
+Paidlancer data are saved in the hard disk automatically exiting the program. There is no need to save manually.
 
 ## FAQ
 
-**Q**: How do I transfer my data to another Computer?<br>
-**A**: Install the app in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous AddressBook home folder.
+**Q**: How do I transfer my data to another Computer?  
+**A**: Install the app in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous TaskGenie home folder.
```

Comment:
TaskGenie? @engenhui1999 Let's change this to PaidLancers!

Reply:
To make sure, is it "PaidLancers" or "Paidlancers"?

Reply:
My bad, `Paidlancers` according to our documents. Thank you for pointing that out!